Pokedex_ios/Services/PokedexService.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  PokedexService.swift
+//  Pokedex_ios
+//
+//  Created by Alvaro Choque on 29/6/22.
+//
+
+import Foundation
+
+protocol PokedexServiceProtocol {
+    func getPokemons(completion: @escaping (_ success: Bool, _ results: [PokemonRaw]?, _ error: String?) -> ())
+}
+
+class PokedexService: PokedexServiceProtocol {
+    func getPokemons(completion: @escaping (Bool, [PokemonRaw]?, String?) -> ()) {
+        
+        // MARK: REFACTOR, Here change the whole function to work with apollo/graphql
```

Comment:
Just for the record (I Know this will be deleted) for future cases use the TODO: tag and don´t add a Mark.